server/app.js

```diff
@@ -1,34 +1,48 @@
-/* eslint-disable import/no-dynamic-require */
-const createError = require('http-errors');
-const express = require('express');
-const path = require('path');
-const fs = require('fs');
-require('dotenv').config();
-const mongoose = require('mongoose');
-const pino = require('express-pino-logger')();
-const logger = require('./utils/logger');
+import createError from 'http-errors';
+import express from 'express';
+import path from 'path';
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+import requireAll from 'require-all';
+import pino from 'express-pino-logger';
+import cors from 'cors';
+import logger from './utils/logger';
+// import authorize from './thirdparty/routes/authorize';
```

Comment:
remove comments if not required :)

Reply:
I have copied code from the hackathon branch, so it has all controllers and routes required for third party integration